lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,114 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # prevent autorun from running tests when the vm closes
+  def self.autorun
+  end
+end
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      # represents the data associated with a test
+      class TestCase
+        attr_accessor :class_name
+        attr_accessor :test_method
+
+        def initialize(class_name, test_method)
+          @class_name = class_name
+          @test_method = test_method
+        end
```

Comment:
`#class_name`, and `#test_method` do not have public call sides, so you can use `include Concord.new(:class_name, :test_method)` to not create the public interface.